phpBB/styles/prosilver/theme/colours.css

```diff
@@ -369,6 +369,41 @@ p.post-notice {
 	background-image: none;
 }
 
+/* colours and backgrounds for mentions.css */
+
+/* mention dropdown */
+.mention-container { /* mention-container */
+	background-color: #ffffff;
+	box-shadow:
+		0 3px 1px -2px rgba(0, 0, 0, 0.2),
+		0 2px 2px 0 rgba(0, 0, 0, 0.14),
+		0 1px 5px 0 rgba(0, 0, 0, 0.12);
+}
+
+.mention-media {
+	color: #757575;
+}
+
+.mention-item {
+	border-bottom-color: #dddddd;
+	color: #212121;
+}
+
+.mention-item:hover,
+.mention-item.cur {
+	background-color: #eeeeee;
+	color: #2d80d2;
+}
+
+.mention-item:hover .mention-media-avatar,
+.mention-item.cur .mention-media-avatar {
```

Comment:
please change all uses of .cur class to .is-active for consistency with standards